iothub/device/src/Transport/AmqpIot/AmqpIotTransport.cs

```diff
@@ -120,14 +122,14 @@ private async Task<TransportBase> CreateClientWebSocketTransportAsync(Cancellati
             }
         }
 
-        private async Task<ClientWebSocket> CreateClientWebSocketAsync(Uri websocketUri, CancellationToken cancellationToken)
+        private ClientWebSocket CreateClientWebSocket()
         {
             try
             {
                 if (Logging.IsEnabled)
-                    Logging.Enter(this, nameof(CreateClientWebSocketAsync));
+                    Logging.Enter(this, nameof(CreateClientWebSocket));
 
-                var websocket = new ClientWebSocket();
+                using var websocket = new ClientWebSocket();
```

Comment:
If you say using, it'll be disposed after we return it. We don't want to do that.